augur/tree.py

```diff
@@ -133,18 +133,29 @@ def build_iqtree(aln_file, out_file, substitution_model="GTR", clean_up=True, nt
         aln_file    file name of input aligment
         out_file    file name to write tree to
     '''
-    with open(aln_file, encoding='utf-8') as ifile:
-        tmp_seqs = ifile.readlines()
+    # create a dictionary for characters that IQ-tree changes.
+    # we remove those prior to tree-building and reinstantiate later
+    def random_string(n):
+        from string import ascii_uppercase as letters
+        return "".join([letters[i] for i in np.random.randint(len(letters), size=n)])
+    prefix = random_string(20)
+    escape_dict = {c:f'_{prefix}-{random_string(20)}_' for c in '/|()*'}
```

Comment:
agreed. this could be anything and 20 random characters should be more than sufficient. I didn't put a lot of thought here, but was thinking that something common to our escapes might make sense. we could replace the prefix by `prefix="DELIM"`. But it doesn't really matter..